src/assets/ckeditor/samples/toolbarconfigurator/lib/codemirror/codemirror.css

```diff
@@ -64,17 +64,17 @@
   -moz-animation: blink 1.06s steps(1) infinite;
   animation: blink 1.06s steps(1) infinite;
 }
-@-moz-keyframes blink {
+@-moz-keyframes codemirror_blink {

```

Comment:
Was this part of the upgrade?

Reply:
Yes, https://angular.io/guide/update-to-version-15#v15-bc-03